core/kazoo_stdlib/src/kz_json.erl

```diff
@@ -257,7 +257,7 @@ are_equal(JObj1, JObj2) ->
 %%------------------------------------------------------------------------------
 -spec from_list([set_value_kv()] | flat_proplist()) -> object() | flat_object().
 from_list(L) when is_list(L) ->
-    ?JSON_WRAPPER(props:filter_undefined(L)).
+    ?JSON_WRAPPER([{K, utf8_binary(V)} || {K, V} <- props:filter_undefined(L)]).
```

Comment:
I would just add the filter here directly rather than calling out to `props`. As-is, you'll traverse the list to filter, then traverse the new list to map the `utf8_binary` function. Just make life easy and `[{K, utf8_binary(V)} || {K, V} <- L, V =/= 'undefined'].` One pass, filter and map :)

Reply:
Nice tip, thanks :)